.github/workflows/update-gpuci.yml

```diff
@@ -0,0 +1,82 @@
+name: Check for gpuCI updates
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch:
+
+jobs:
+  update-gpuci:
+    runs-on: ubuntu-latest
+    if: github.repository == 'rapidsai/dask-build-environment'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get old cuDF / UCX-Py versions
+        run: |
+          echo RAPIDS_VER=$(grep -A 2 "RAPIDS_VER" ci/axis/dask.yaml | tail -n1 | grep -o "'.*'" | sed "s/'//g") >> $GITHUB_ENV
+          echo UCX_PY_VER=$(grep -m2 "UCX_PY_VER" ci/gpuci/run.sh | tail -n1 | grep -o '".*"' | sed 's/"//g') >> $GITHUB_ENV
+
+      - name: Get latest cuDF nightly version
+        id: cudf_latest
+        uses: jacobtomlinson/gha-anaconda-package-version@0.1.3
+        with:
+          org: "rapidsai-nightly"
+          package: "cudf"
+          version_system: "CalVer"
+
+      - name: Get latest UCX-Py nightly version
+        id: ucx_py_latest
+        uses: jacobtomlinson/gha-anaconda-package-version@0.1.3
+        with:
+          org: "rapidsai-nightly"
+          package: "ucx-py"
+          version_system: "CalVer"
+
+      - name: Strip git tags from versions
+        env:
+          FULL_RAPIDS_VER: ${{ steps.cudf_latest.outputs.version }}
+          FULL_UCX_PY_VER: ${{ steps.ucx_py_latest.outputs.version }}
+        run: |
+          echo "NEW_RAPIDS_VER=${FULL_RAPIDS_VER::-10}" >> $GITHUB_ENV
+          echo "NEW_UCX_PY_VER=${FULL_UCX_PY_VER::-9}" >> $GITHUB_ENV
+
+      - name: Update axis file
+        uses: jacobtomlinson/gha-find-replace@0.1.4
+        with:
+          include: 'ci\/axis\/dask\.yaml'
```

Comment:
We might want to switch to `v2` here and use globs instead of regex. Makes this a little cleaner.

Reply:
Thanks! I'll also update this in https://github.com/dask/dask/blob/main/.github/workflows/update-gpuci.yml

Reply:
I ended up restructuring the find-replace steps to avoid regex altogether - should make it a little easier to revisit later on